### DISCUSSION TITLE
ChatGPT-4 대신 물어봐드립니다.

### DISCUSSION THREAD

**Discussion by ccmuk58** [The-Day-In-1984/1984_Project#95] [Show and tell]
![image](https://user-images.githubusercontent.com/29997371/230917514-aecb47fe-fdc4-49ed-9fa0-47ef4aae159e.png)

쓰레드 달아주면 대신 물어봐줍니다 ^_^

디스코드나 카톡도 가능

짧은 시간에 연속적으로 사용하고 싶으면 따로 연락 ^-^

**Comment by fkdl0048** [comment]
Chat 장현T 1984스토리를 이용한 스토리 게임의 게임명을 추천해줘 (10개 정도)

**Reply by ccmuk58** [reply]
1. "Big Brother's Watch"
2. "Orwellian Odyssey"
3. "Thoughtcrime Tales"
4. "Dystopia Unraveled"
5. "Doublethink Dilemma"
6. "Newspeak Chronicles"
7. "Oceania Unleashed"
8. "Room 101 Reckoning"
9. "Proles' Uprising"
10. "Ingsoc Insurgence"

**Reply by fkdl0048** [reply]
여기 맛집이네요..?
